src/MPDevice.cpp

```diff
@@ -1287,6 +1293,53 @@ void MPDevice::memMgmtModeReadFlash(AsyncJobs *jobs, bool fullScan,
     }
 }
 
+//TODO: Remove if favorites, cpz_ctr, ctrvalue and getData implemented for BLE too
+void MPDevice::memMgmtModeReadFlashBLE(AsyncJobs *jobs, bool fullScan, const MPDeviceProgressCb &cbProgress, bool getCreds)
+{
+    if (getCreds)
+    {
+        /* Get parent node start address */
+        jobs->append(new MPCommandJob(this, MPCmd::GET_STARTING_PARENT,
```

Comment:
to be fair it should be called get starting parents :)

Reply:
Now yes, but later it will be deleted. 
And during implementing the commands I will copy the other jobs from `memMgmtModeReadFlash` to here for testing it. :)

Reply:
ok!